.ci/test_daemon.py

```diff
@@ -91,9 +91,9 @@ def validate_calculations(expected_results):
             valid = False
 
         try:
-            actual_dict = calc.out.output_parameters.get_dict()
+            actual_dict = calc.out.parameters.get_dict()
```

Comment:
Apparently, after discussion with some users, `parameters` conveys a meaning of being an input. `output_parameters` was alleviating partially this issue (even if ParameterData is still not necessarily the good name). This change however makes things even less clear (you risk to refer to these as the 'parameters' of the calculation, making it ambiguous). Maybe we can find a different name? E.g. results, for the default output? Or some other better name?

We should also think if we want to rename ParameterData e.g. to Dict and move it into 'base', but this is probably out of scope for this PR.

Reply:
I just don't like the superfluous `output_` since it is already an output, but I agree, `parameters` is also pretty bad. Clearly, I am all for changing the name. I would be all for changing `ParameterData` into `Dict` and subsequently calling the default output `results`. This is one of the things I wanted to discuss with the other plugin developers, in coming up with a common interface. Shall I revert this change and we do everything in one go? But I guess, if we are going to change `ParameterData`, we should do it now.